payments_payu/provider.py

```diff
@@ -43,16 +43,20 @@
 
 
 def add_extra_data(payment, new_extra_data):
+    payment.refresh_from_db()
```

Comment:
Wouldn't this be better?

```suggestion
    payment.refresh_from_db(fields=["extra_data"])
```

Just to make it more future-proof...